packages/salesforcedx-vscode-core/src/commands/forceSourceDiff.ts

```diff
@@ -5,15 +5,23 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
+import { StreamingMockSubscriptionCall } from '@salesforce/core/lib/testSetup';
 import {
   CliCommandExecutor,
   Command,
   DiffResultParser,
   SfdxCommandBuilder
 } from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
 import { ContinueResponse } from '@salesforce/salesforcedx-utils-vscode/out/src/types';
+import * as path from 'path';
 import * as vscode from 'vscode';
 import { channelService } from '../channels';
+import * as conflictDetectionService from '../conflict/conflictDetectionService';
+import {
+  MetadataCacheExecutor,
+  MetadataCacheResult
+} from '../conflict/metadataCacheService';
```

Comment:
can we export these three at the top level from the `conflict` folder?

Reply:
Moved these and the others to the top level.